### PR TITLE
Update SDKs to send clicks, change default test space

### PR DESCRIPTION
**Pull request by msub2**
This updates all web integrations and Unity to send click events to the beacon server, as well as updating the default space in the test pages so it isn't on an in-use space. I also rolled the beacon script in Unity into the banner script, as it was honestly too small to warrant its own file and works better just being part of the banner script.

### CODE REVIEW

File: tests/aframe/index.html [hunk 1]
```diff
@@ -11,46 +11,46 @@
             <a-entity 
             visible
             id="banner1"
-            zesty-banner="space: 0; creator: 0x0000000000000000000000000000000000000000; format: tall; style: standard; height: 2;" 
+            zesty-banner="space: 999; creator: 0x0000000000000000000000000000000000000000; format: tall; style: standard; height: 2; beacon: true"
```
Comment (limbofeather) [inline tests/aframe/index.html:14]: I have a feeling we might hit space 999 at some point - should we go for a higher number?

**Review by limbofeather — APPROVED** [review]
LGTM, just one small note about the testing space ID.

File: tests/aframe/index.html (hunk 1 above)
Comment (msub2) [reply tests/aframe/index.html:14]: I don't expect we're going to hit it anytime soon, but if you want to bump it up now we could go for like 9999

Comment (limbofeather) [reply tests/aframe/index.html:14]: Go for like 999999

**Pull request merged**